### PR TITLE
fix(TDP-3551): add a message when recipe is empty

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3551

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/411/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/411/)

File: dataprep-webapp/src/app/components/recipe/recipe.html [hunk 1]
```diff
@@ -24,6 +24,10 @@
      top-delta="0"
      while-dragging="$ctrl.isDragStart"
 >
+    <div class="empty-message"
+         ng-if="$ctrl.state.playground.recipe.current.reorderedSteps.length == 0"
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/recipe/recipe.html]: === instead?

File: dataprep-webapp/src/app/components/recipe/_recipe.scss [hunk 2]
```diff
@@ -24,6 +24,15 @@ $recipe-step-draggable-background-color: #808080 !default;
   padding-bottom: 10px;
   width: 100%;
 
+  .empty-message {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    width: 100%;
+    justify-content: center;
+    padding: 20px;
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/recipe/_recipe.scss]: why?

Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/recipe/_recipe.scss]: image

Does not work on Safari

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/423/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/423/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/424/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/424/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/442/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/442/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/443/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/443/)

**Pull request merged**